### ISSUE TITLE
#a11y: Incorrect initial focus position when opening Settings

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/desktop#1333]
### Steps To Reproduce

1. Go to "File > Settings" - the settings dialog appears
2. `Tab` forward

### Expected Result

Keyboard focus should be at the top of the modal dialog when it opens. First `Tab` should then go to the first focusable control in the modal.

### Actual Result

First `Tab` goes to "Enable Tray Icon" in "App Settings (All Accounts)", in the middle of the modal dialog.

### Screenshots or Videos

Animated GIF showing the File > Settings menu, and once the modal is opened, a `Tab` forward and two more tabs after that.
![bitwarden-desktop-settings-initial-focus](https://user-images.githubusercontent.com/895831/153775065-bd1f1089-732c-4e30-8ecb-357bfaa97e2b.gif)

### Operating System

Windows

### Operating System Version

10

### Installation method

Direct Download (from bitwarden.com)

### Build Version

Version 1.31.2 Shell 16.0.7 Renderer 96.0.4664.110 Node 16.9.1 Architecture x64

**Comment by eliykat** [comment]
Quick debug: reproducible. We do autofocus the first tabbable element in dynamic modals, but the Security and Preferences sections take a moment to load in (due to async calls), meaning that their controls are missed when figuring out the "first tabbable element". Will need to figure out a less brittle way of dealing with this.

Thanks for the report @patrickhlauke
